electron_repulsion/direct.py

```diff
@@ -40,7 +40,7 @@ def get_atom_string(atoms, locs):
     return atom_string, str
 
 def num_tiles():
-        return 1472
+        return int(os.getenv('JAX_IPU_MODEL_NUM_TILES', 1472))

```

Comment:
I believe it is cleaner here to use `jax` device interface, i.e. `jax.devices("ipu")[0].num_tiles`

Reply:
good point.